src/components/auth/index.ts

```diff
@@ -1,2 +0,0 @@
-export * from './auth.service';
-export * from './auth.module';
```

Comment:
Let's keep using index files for exports.
They help convey which exports are "public" vs private to the module.
If anything, to be consistent with the rest of the codebase.

Reply:
I believe that this was a change made by Michael based on a prior discussion w/ Daniel regarding circular dependency issues in node that arise from using index files for batch exports. Happy to bring it back though if the consensus was to resume the use of index files?

Reply:
That was an edge case. Using the index.ts files was just the symptom. We shouldn't have a problem.